boards/nordic/bm_nrf54l15dk/bm_nrf54l15dk_nrf54l15_cpuapp_s115_softdevice_mcuboot.dts

```diff
@@ -97,6 +97,11 @@
 &cpuapp_sram {
 	status = "okay";
 
+	/* trim down to not overlap with retainedmem */
```

Comment:
/* Override setting in bm_nrf54l15dk_nrf54l15_cpuapp_common.dtsi to adjust sram size to not overlap with retainedmem */